tests/integration/test.prefix.js

```diff
@@ -38,7 +38,9 @@ describe('test.prefix.js', function () {
 if (typeof process !== 'undefined' &&
     !process.env.LEVEL_ADAPTER &&
     !process.env.LEVEL_PREFIX &&
-    !process.env.ADAPTER) {
+    !process.env.ADAPTER &&
+    // fails on windows with EBUSY - "resource busy or locked", not worth fixing
```

Comment:
Hey @nolanlawson quick heads up... were you using VS Code here when you received the dreaded ` EBUSY - "resource busy or locked"`?

I've experience this (and similar) issues with Windows / Node dev when using VS Code. I found (with a handy little utility _Process Explorer_) that VS Code was the culprit. When I close VS Code and run my scripts, everything works.

YMMV ;)